tests/rule_based_profiler/parameter_builder/test_simple_date_format_string_parameter_builder.py

```diff
@@ -13,6 +13,9 @@
     ParameterContainer,
     get_parameter_value_by_fully_qualified_parameter_name,
 )
+from great_expectations.rule_based_profiler.util import (
+    get_parameter_value_and_validate_return_type,
+)
```

Comment:
All changes in `test_simple_date_format_string_parameter_builder` are an unrelated refactor to add type checking to previous work.